src/open_inwoner/accounts/tests/test_auth.py

```diff
@@ -68,7 +69,11 @@ def test_registration_page_only_digid_with_invite(self):
             furl(reverse("digid:login")).add({"next": necessary_url}).url,
         )
 
-    def test_digid_fail_without_invite_redirects_to_login_page(self):
+    @patch("digid_eherkenning.validators.Proef11ValidatorBase.__call__")
+    def test_digid_fail_without_invite_redirects_to_login_page(self, m):
+        # disable mock form validation to check redirect
+        m.return_value = True
```

Comment:
Why do we need to mock the validator? Couldn't we just use a valid 11-proef test value?

Reply:
We are testing the redirect for invalid BSN numbers. However, a BSN can be invalid for two reasons: (a) it fails formal validation, (b) it is formally valid but doesn't correspond to an actual BSN. If we pass a valid 11-proef value, we won't be able to test the redirect as we can't test actual BSN's.